packages/carousel/.storybook/main.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  addons: [
+    {
+      name: '@pluralsight/ps-design-system-storybook-preset',
+      options: { center: false }
```

Comment:
i've setup the preset so we can disable addons as needed. there are some packages that don't look good when centered. i've disabled the centering for the stories i could find - hopefully i didn't miss any